packages/api/src/external/carequality/api.ts

```diff
@@ -11,3 +12,20 @@ export function makeCarequalityAPI(apiKey?: string): Carequality | undefined {
   const cqApiKey = apiKey ?? Config.getCQApiKey();
   return new Carequality(cqApiKey);
 }
+
+/**
+ * Creates a new instance of the IHE Gateway client.
+ * @returns IHE Gateway client.
+ */
+export function makeIheGatewayAPI(): IHEGateway {
+  // TODO ADD IHE GATEWAY MOCK
+  // if (Config.isSandbox()) {
+  //   return;
+  // }
+
+  if (Config.isDev() || Config.isStaging()) {
+    return new IHEGateway(APIMode.dev);
```

Comment:
dev will point to a different endpoint vs staging, right?

dev is local Mirth, staging will be on AWS Mirth